src/site/content/en/blog/cds2021-updates/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Everything announced at Chrome Dev Summit 2021
+subhead: A roundup of all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/gK7MVviHglsjJNEDG6Au.png
+alt: Chrome Dev Summit 2021
+authors:
+  - rachelandrew
+description: >
+  A roundup of all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
+date: 2021-11-03
+tags:
+  - blog
+---
+
+TO DO: add recap video
+
+This post rounds up all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
+
+## Make the web more interoperable 
+
+Web compatibility is one of the biggest challenges for web developers, 
+and so Google has been working with other browser vendors to fix the top five compatibility pain points:  
+[flexbox](/learn/css/flexbox/), 
+[grid](/learn/css/grid/), 
+position: sticky, 
+[aspect-ratio](/aspect-ratio/), 
+and [transforms](/learn/css/transforms/).
```

Comment:
This link doesn't work, there's this though `learn/css/transitions/#transform`

Reply:
```suggestion
and [transforms](/learn/css/transitions/#transform).
```